### DISCUSSION TITLE
Support for dedicated IP feature of PIA?

### DISCUSSION THREAD

**Discussion by richardtheb** [haugene/docker-transmission-openvpn#2210] [Q&A]
Hey, PIA offers a dedicated IP address feature that I would like to use. Does the docker support this? I've got it connected, but there is not a .ovpn file for it.

**Comment by Iron-Juice** [comment]
@richardtheb  were you able to get this resolved? I have tried to use the OPENVPN_CONFIG env variable, but that doesn't work out.

**Comment by stevenboyd78** [comment]
Any possibility of this being implimented?
